collector/main.py

```diff
@@ -1,10 +1,12 @@
 import argparse
+import base64
 import logging
 import sys
 import time
+import os
 
 import requests
-
+from dotenv import load_dotenv
```

Comment:
I would rather use `os.environ` than another library so we can keep the dependencies small.

Reply:
this is just if people want to use .env files. so they can mount their .env file or use it.